apps/client/src/components/new_toast/new_toast.module.css

```
@@ -24,12 +29,13 @@ html, body {
     letter-spacing: 0.3rem;
     text-align: center;
 }
+
 .form {
     display: flex;
     flex-direction: column;
     align-items: center; 
     width: 100%;
-    max-width: 35rem;
+    max-width: 50rem;
     padding: 3rem;
     background: rgba(255, 255, 255, 0.95);
```

```suggestion


    background: rgba(255, 255, 255, 0.95);
```.  add var() css to rgba